### ISSUE TITLE
Crash while closing ChatActivity

### ISSUE THREAD

**Issue by FrantisekGazo** [crisp-im/crisp-sdk-android#57]
I see couple of crashes like this in our crashlitics system.
From additional data it looks like it happend when ChatActivity was closing.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
    at im.crisp.client.b.d.d.b.i
    at im.crisp.client.b.d.d.b.b
    at im.crisp.client.b.e.b.a.onStop
    at androidx.fragment.app.Fragment.performStop(Fragment.java:2797)
    at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:940)
    at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
    at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
    at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:2659)
    at androidx.fragment.app.FragmentManagerImpl.dispatchStop(FragmentManagerImpl.java:2634)
    at androidx.fragment.app.FragmentController.dispatchStop(FragmentController.java:290)
    at androidx.fragment.app.FragmentActivity.onStop(FragmentActivity.java:564)
    at androidx.appcompat.app.AppCompatActivity.onStop(AppCompatActivity.java:216)
    at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1277)
    at android.app.Activity.performStop(Activity.java:6185)
    at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3670)
    at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3729)
    at android.app.ActivityThread.access$1400(ActivityThread.java:152)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1358)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:135)
    at android.app.ActivityThread.main(ActivityThread.java:5259)
    at java.lang.reflect.Method.invoke(Method.java)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:902)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:697)

java.lang.RuntimeException: Unable to stop activity {com.ventrata.app/im.crisp.client.ChatActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
    at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3675)
    at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3729)
    at android.app.ActivityThread.access$1400(ActivityThread.java:152)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1358)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:135)
    at android.app.ActivityThread.main(ActivityThread.java:5259)
    at java.lang.reflect.Method.invoke(Method.java)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:902)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:697)
```

**Comment by eliottvincent** [comment]
Hey, we released a new version (1.0.7) that should fix this bug, can you please try?

**Comment by FrantisekGazo** [comment]
Thanks.
will reopen if I see it happen on new version.
